[2019년04월23일 10시24분29초] 
 Exception 
Error Message : java.lang.ArrayIndexOutOfBoundsException: 1
	at risk.dashboard.realstate.RealstateMgr.getOnlineTrend(RealstateMgr.java:106)
	at org.apache.jsp.dashboard.realstate.realstateExcelDao_jsp._jspService(realstateExcelDao_jsp.java:119)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:388)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:321)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:267)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:610)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:503)
	at java.lang.Thread.run(Unknown Source)

[2019년04월23일 10시24분51초] 
 Exception 
Error Message : java.lang.ArrayIndexOutOfBoundsException: 1
	at risk.dashboard.realstate.RealstateMgr.getOnlineTrend(RealstateMgr.java:106)
	at org.apache.jsp.dashboard.realstate.realstateExcelDao_jsp._jspService(realstateExcelDao_jsp.java:119)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:388)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:321)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:267)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:610)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:503)
	at java.lang.Thread.run(Unknown Source)

[2019년04월23일 10시25분25초] 
 Exception 
Error Message : java.lang.ArrayIndexOutOfBoundsException: 1
	at risk.dashboard.realstate.RealstateMgr.getOnlineTrend(RealstateMgr.java:106)
	at org.apache.jsp.dashboard.realstate.realstateExcelDao_jsp._jspService(realstateExcelDao_jsp.java:119)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:388)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:321)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:267)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:610)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:503)
	at java.lang.Thread.run(Unknown Source)

[2019년04월23일 10시25분31초] 
 Exception 
Error Message : java.lang.ArrayIndexOutOfBoundsException: 1
	at risk.dashboard.realstate.RealstateMgr.getOnlineTrend(RealstateMgr.java:106)
	at org.apache.jsp.dashboard.realstate.realstateExcelDao_jsp._jspService(realstateExcelDao_jsp.java:119)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:388)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:321)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:267)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:610)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:503)
	at java.lang.Thread.run(Unknown Source)

[2019년04월23일 10시26분05초] 
 Exception 
Error Message : java.lang.ArrayIndexOutOfBoundsException: 1
	at risk.dashboard.realstate.RealstateMgr.getOnlineTrend(RealstateMgr.java:106)
	at org.apache.jsp.dashboard.realstate.realstateExcelDao_jsp._jspService(realstateExcelDao_jsp.java:119)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:388)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:321)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:267)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:610)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:503)
	at java.lang.Thread.run(Unknown Source)

[2019년04월23일 10시26분14초] 
 Exception 
Error Message : java.lang.ArrayIndexOutOfBoundsException: 1
	at risk.dashboard.realstate.RealstateMgr.getOnlineTrend(RealstateMgr.java:106)
	at org.apache.jsp.dashboard.realstate.realstateExcelDao_jsp._jspService(realstateExcelDao_jsp.java:119)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:388)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:321)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:267)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:610)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:503)
	at java.lang.Thread.run(Unknown Source)

[2019년04월23일 10시39분47초] 
 SQLException
Oracle Error Code [1064]
SQLState          [42000]
<<-----------------------------------------------------------------------
 SELECT  																						
        AA.NAME 																					
       ,AA.S_SEQ 																				
       ,IFNULL(BB.CNT,0) AS CNT 																	
 FROM  																							
 ( 																								
 SELECT  																						
      '2300475' AS S_SEQ 																		
     ,'블로그' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300476' AS S_SEQ 																			
     ,'트위터' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300477' AS S_SEQ 																			
     ,'유튜브' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300478' AS S_SEQ 																			
     ,'인스타그램' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300598' AS S_SEQ 																			
     ,'카카오스토리' AS NAME 																		
 UNION ALL 																						
 SELECT  																						
     '2300599' AS S_SEQ 																			
     ,'카페' AS NAME 																				
 UNION ALL 																						
 SELECT  																						
     '2300479' AS S_SEQ 																			
     ,'페이스북' AS NAME 																			
 ) AA LEFT OUTER JOIN 																			
 ( 																								
 SELECT  																						
 			  A.S_SEQ                	 														
        ,COUNT(0) AS CNT 																		
 FROM ISSUE_DATA A , ISSUE_DATA_CODE C ,ISSUE_DATA_CODE B										
	WHERE A.MD_DATE BETWEEN '2019-04-17 00:00:00' AND '2019-04-23 23:59:59' 							
   AND A.ID_SEQ = C.ID_SEQ 																		
   AND C.IC_TYPE = 6 																			
   AND C.IC_CODE = 10  																			
	  AND A.ID_SEQ = B.ID_SEQ 																		
			  	AND B.IC_TYPE = 				 													
 GROUP BY A.S_SEQ     																			
 )BB ON AA.S_SEQ = BB.S_SEQ  																	

----------------------------------------------------------------------->>
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'GROUP BY A.S_SEQ     																			
 )BB ON AA.S_SEQ = BB.S_SEQ' at line 46
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.Util.getInstance(Util.java:381)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1031)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3376)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3308)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1837)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1961)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2543)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1737)
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1888)
	at org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	at org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	at risk.dashboard.media.MediaMgr.getChannelInfo(MediaMgr.java:244)
	at org.apache.jsp.dashboard.media.mediaExcelDao_jsp._jspService(mediaExcelDao_jsp.java:191)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:388)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:321)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:267)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:610)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:503)
	at java.lang.Thread.run(Unknown Source)

[2019년04월23일 10시40분14초] 
 SQLException
Oracle Error Code [1064]
SQLState          [42000]
<<-----------------------------------------------------------------------
 SELECT  																						
        AA.NAME 																					
       ,AA.S_SEQ 																				
       ,IFNULL(BB.CNT,0) AS CNT 																	
 FROM  																							
 ( 																								
 SELECT  																						
      '2300475' AS S_SEQ 																		
     ,'블로그' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300476' AS S_SEQ 																			
     ,'트위터' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300477' AS S_SEQ 																			
     ,'유튜브' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300478' AS S_SEQ 																			
     ,'인스타그램' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300598' AS S_SEQ 																			
     ,'카카오스토리' AS NAME 																		
 UNION ALL 																						
 SELECT  																						
     '2300599' AS S_SEQ 																			
     ,'카페' AS NAME 																				
 UNION ALL 																						
 SELECT  																						
     '2300479' AS S_SEQ 																			
     ,'페이스북' AS NAME 																			
 ) AA LEFT OUTER JOIN 																			
 ( 																								
 SELECT  																						
 			  A.S_SEQ                	 														
        ,COUNT(0) AS CNT 																		
 FROM ISSUE_DATA A , ISSUE_DATA_CODE C ,ISSUE_DATA_CODE B										
	WHERE A.MD_DATE BETWEEN '2019-04-17 00:00:00' AND '2019-04-23 23:59:59' 							
   AND A.ID_SEQ = C.ID_SEQ 																		
   AND C.IC_TYPE = 6 																			
   AND C.IC_CODE = 10  																			
	  AND A.ID_SEQ = B.ID_SEQ 																		
			  	AND B.IC_TYPE = 				 													
 GROUP BY A.S_SEQ     																			
 )BB ON AA.S_SEQ = BB.S_SEQ  																	

----------------------------------------------------------------------->>
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'GROUP BY A.S_SEQ     																			
 )BB ON AA.S_SEQ = BB.S_SEQ' at line 46
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.Util.getInstance(Util.java:381)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1031)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3376)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3308)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1837)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1961)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2543)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1737)
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1888)
	at org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	at org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	at risk.dashboard.media.MediaMgr.getChannelInfo(MediaMgr.java:244)
	at org.apache.jsp.dashboard.media.mediaExcelDao_jsp._jspService(mediaExcelDao_jsp.java:191)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:388)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:321)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:267)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:610)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:503)
	at java.lang.Thread.run(Unknown Source)

[2019년04월23일 10시40분22초] 
 SQLException
Oracle Error Code [1064]
SQLState          [42000]
<<-----------------------------------------------------------------------
 SELECT  																						
        AA.NAME 																					
       ,AA.S_SEQ 																				
       ,IFNULL(BB.CNT,0) AS CNT 																	
 FROM  																							
 ( 																								
 SELECT  																						
      '2300475' AS S_SEQ 																		
     ,'블로그' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300476' AS S_SEQ 																			
     ,'트위터' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300477' AS S_SEQ 																			
     ,'유튜브' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300478' AS S_SEQ 																			
     ,'인스타그램' AS NAME 																			
 UNION ALL 																						
 SELECT  																						
     '2300598' AS S_SEQ 																			
     ,'카카오스토리' AS NAME 																		
 UNION ALL 																						
 SELECT  																						
     '2300599' AS S_SEQ 																			
     ,'카페' AS NAME 																				
 UNION ALL 																						
 SELECT  																						
     '2300479' AS S_SEQ 																			
     ,'페이스북' AS NAME 																			
 ) AA LEFT OUTER JOIN 																			
 ( 																								
 SELECT  																						
 			  A.S_SEQ                	 														
        ,COUNT(0) AS CNT 																		
 FROM ISSUE_DATA A , ISSUE_DATA_CODE C ,ISSUE_DATA_CODE B										
	WHERE A.MD_DATE BETWEEN '2019-04-17 00:00:00' AND '2019-04-23 23:59:59' 							
   AND A.ID_SEQ = C.ID_SEQ 																		
   AND C.IC_TYPE = 6 																			
   AND C.IC_CODE = 10  																			
	  AND A.ID_SEQ = B.ID_SEQ 																		
			  	AND B.IC_TYPE = 				 													
 GROUP BY A.S_SEQ     																			
 )BB ON AA.S_SEQ = BB.S_SEQ  																	

----------------------------------------------------------------------->>
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'GROUP BY A.S_SEQ     																			
 )BB ON AA.S_SEQ = BB.S_SEQ' at line 46
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.Util.getInstance(Util.java:381)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1031)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3376)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3308)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1837)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1961)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2543)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1737)
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1888)
	at org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	at org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	at risk.dashboard.media.MediaMgr.getChannelInfo(MediaMgr.java:244)
	at org.apache.jsp.dashboard.media.mediaExcelDao_jsp._jspService(mediaExcelDao_jsp.java:191)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:388)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:321)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:267)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:723)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:293)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:859)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:610)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:503)
	at java.lang.Thread.run(Unknown Source)

